tests/phpunit/api/v3/LoggingTest.php

```diff
@@ -346,7 +346,7 @@ public function testRevertNoDate() {
   public function testRevertRestrictedTables() {
 
     CRM_Core_DAO::executeQuery("SET @uniqueID = 'temp name'");
-    $this->callAPISuccessGetValue('Setting', array('name' => 'logging_all_tables_uniquid'), TRUE);
+    $this->assertEquals(TRUE, $this->callAPISuccessGetValue('Setting', ['name' => 'logging_all_tables_uniquid']));
```

Comment:
There's also the `assertTrue` function :)

Reply:
$this-assertTrue(exists('assertTrue'))